test/symmetric_difference.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright Louis Dionne 2013-2017
```

Comment:
Same comment as for examples; it would be nice to have a test for map and a test for sets in different files.